ethcore/src/client/test_client.rs

```diff
@@ -352,7 +352,7 @@ pub fn get_temp_state_db() -> GuardedTempResult<StateDB> {
 
 impl MiningBlockChainClient for TestBlockChainClient {
 	fn latest_schedule(&self) -> Schedule {
-		Schedule::new_post_eip150(24576, true, true, true)
```

Comment:
IMHO this is reaching a point where a struct would look much better than the 4 boolean flags.
```rust
Schedule::new_post_eip150(PostEip150 {
   max_code_size: 24576,
   fix_exp: true,
   no_empty: true,
   kill_empty: true,
   static_call: true,
});
```